apps/server/src/api-data/automation/automation.controller.ts

```
@@ -0,0 +1,118 @@
+      title: req.body.title ?? undefined,
+      trigger: req.body.trigger ?? undefined,
+      blueprintId: req.body.blueprintId ?? undefined,
+    });
+    res.status(200).send(automation);
+  } catch (error) {
+    const message = getErrorMessage(error);
+    res.status(400).send({ message });
+  }
+}
+
+export function deleteAutomation(req: Request, res: Response<void | ErrorResponse>) {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)